docs/technical-details/release-notes/v4-tucker/4.5.22.md

```diff
@@ -0,0 +1,12 @@
+---
+title: 4.5.22
+sidebar_position: 59478
```

Comment:
need to decrement until we have some other way to sort the release notes in the sidebar

Reply:
<img width="285" height="165" alt="image" src="https://github.com/user-attachments/assets/cd2be988-1140-4e91-8213-f437f19e5cf1" />

Reply:
#249